aio/tools/transforms/angular-base-package/index.js

```diff
@@ -5,17 +5,21 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+const { extname, resolve } = require('canonical-path');
+const { existsSync } = require('fs');
 const path = require('path');
-const Package = require('dgeni').Package;
 
+const Package = require('dgeni').Package;
 const gitPackage = require('dgeni-packages/git');
 const jsdocPackage = require('dgeni-packages/jsdoc');
 const nunjucksPackage = require('dgeni-packages/nunjucks');
+const postProcessPackage = require('dgeni-packages/post-process-html');
+
+const { SRC_PATH } = require('../config');
 const linksPackage = require('../links-package');
 const examplesPackage = require('../examples-package');
 const targetPackage = require('../target-package');
 const remarkPackage = require('../remark-package');
```

Comment:
Since you're ordering imports alphabetically :grin:

```suggestion
const remarkPackage = require('../remark-package');
const targetPackage = require('../target-package');
```